docs/blog/2024-11-08-chicory-1.0.0-M1.md

```diff
@@ -285,8 +285,8 @@ calling a native executable.
           <configuration>
             <!-- Translate the Wasm binary `wat2wasm` into bytecode -->
             <wasmFile>src/main/resources/wat2wasm</wasmFile>
-            <!-- Generate the following class file as a result -->
-            <name>com.dylibso.chicory.wabt.Wat2WasmModule</name>
+            <!-- Generate the result classes with this prefix -->
+            <name>com.dylibso.chicory.wabt.Wat2Wasm</name>
```

Comment:
Instead of generating two classes with one static method each, can we create 1 class with 2 static methods?

Reply:
That's a good idea. I'll update it.